### DISCUSSION TITLE
Installation Issue

### DISCUSSION THREAD

**Discussion by tomzac19** [calliope-project/calliope#486] [Q&A]
Hello dear,
I'm encountering strong problems when downloading Calliope through the suggested procedure...
The error I get after almost an hour of "solving environments.." is the following: 

`CondaMemoryError: The conda process ran out of memory. Increase system memory and/or try again.`

Do you suggest other ways to install Calliope? Thank you very much!

**Comment by brynpickering** [comment]
Hi @tomzac19 I'm going to close this in favour of the issue #475. Please refer to that issue for possible approaches to a solution.
